full-service/src/json_rpc/txo.rs

```diff
@@ -11,17 +11,15 @@ use serde_derive::{Deserialize, Serialize};
 /// different statuses and types in relation to those accounts.
 #[derive(Deserialize, Serialize, Default, Debug, Clone)]
 pub struct Txo {
-    /// String representing the object's type. Objects of the same type share
-    /// the same value.
-    pub object: String,
```

Comment:
Can we move all the object deletion to another PR?

Reply:
yes, will clean the PR up and move all unnecessary changes to other pr(s)